src/Avalonia.Visuals/RoundedRect.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+
+namespace Avalonia
+{
+    public struct RoundedRect
+    {
+        public bool Equals(RoundedRect other)
+        {
+            return Rect.Equals(other.Rect) && RadiiTopLeft.Equals(other.RadiiTopLeft) && RadiiTopRight.Equals(other.RadiiTopRight) && RadiiBottomLeft.Equals(other.RadiiBottomLeft) && RadiiBottomRight.Equals(other.RadiiBottomRight);
+        }
+
+        public override bool Equals(object obj)
+        {
+            return obj is RoundedRect other && Equals(other);
+        }
+
+        public override int GetHashCode()
+        {
+            unchecked
+            {
+                var hashCode = Rect.GetHashCode();
+                hashCode = (hashCode * 397) ^ RadiiTopLeft.GetHashCode();
+                hashCode = (hashCode * 397) ^ RadiiTopRight.GetHashCode();
+                hashCode = (hashCode * 397) ^ RadiiBottomLeft.GetHashCode();
+                hashCode = (hashCode * 397) ^ RadiiBottomRight.GetHashCode();
+                return hashCode;
```

Comment:
Avalonia works on top of .NetStandard2.0, so it should be possible to use HashCode struct (with [nuget package](https://www.nuget.org/packages/Microsoft.Bcl.HashCode)) to replace code above with
```csharp
return HashCode.Combine(Rect, RadiiTopLeft, RadiiTopRight, RadiiBottomLeft, RadiiBottomRight);
```
So is it worst to use that package at least for new code? Though I am not sure about performance.

Reply:
This is consistent with other overrides.